niworkflows/interfaces/nibabel.py

```diff
@@ -89,3 +89,68 @@ def _run_interface(self, runtime):
         maskimg.to_filename(self._results['out_mask'])
 
         return runtime
+
+
+class _FourToThreeInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='input 4d image')
+
+
+class _FourToThreeOutputSpec(TraitedSpec):
+    out_files = OutputMultiPath(File(exists=True),
+                                     desc='output list of 3d images')
+
+
+class FourToThree(SimpleInterface):
+    """Split a 4D dataset along the last dimension
+    into multiple 3D volumes."""
+
+    input_spec = _FourToThreeInputSpec
+    output_spec = _FourToThreeOutputSpec
+
+    def _run_interface(self, runtime):
+        filenii = nb.load(self.inputs.in_file)
+        if len(filenii.shape) != 4:
+            raise RuntimeError('Input image (%s) is not 4D.' % filenii)
+
+        files_3d = nb.four_to_three(filenii)
+        out_files = []
+        for i, file_3d in enumerate(files_3d):
+            out_file = fname_presuffix(in_file, suffix="_tmp_{}".format(i))
+            file_3d.to_filename(out_file)
+            out_files.append(out_file)
+
+        self._results['out_files'] = out_files
+        return runtime
+
+
+class _ConcatImagesInputSpec(BaseInterfaceInputSpec):
+    in_files = InputMultiPath(File(exists=True, mandatory=True,
+                                   desc='input list of 3d images'))
+
+
+class _ConcatImagesOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='output list of 3d images')
+
+
+class ConcatImages(SimpleInterface):
```

Comment:
Thinking about this, I'm not sure `ConcatImages` is a good name, since you're doing some preprocessing before passing to `concat_images`. `concat_images` is perfectly happy to concatenate along any axis and along that axis there is no requirement that the images have the same dimension.

Reply:
hmm maybe something like `4dMergeImages` would be better?

Reply:
or `FourDMerge`, `MergeImages`, `MergeThreeToFour` ...

Reply:
`Merge` seems fine. Or `MergeImages` or `MergeSeries` if that seems non-specific.

Reply:
`MergeSeries`!